extensions/amp-pan-zoom/1.0/component-ts.tsx

```diff
@@ -0,0 +1,267 @@
+import {
+  GestureHandlers,
+  UserGestureConfig,
+  UserHandlers,
+  useGesture,
+} from '@use-gesture/react';
```

Comment:
If I remember correctly, we were going to try with the Gesture lib? Was that not workable?